package.json

```diff
@@ -13,7 +13,13 @@
     "type": "git",
     "url": "git://github.com/jaegertracing/jaeger-client-node.git"
   },
-  "contributors": ["Yuri Shkuro <ys@uber.com>", "Won Jun Jang <wjang@uber.com>"],
+  "files": [
+    "dist/src"
+  ],
```

Comment:
is this the whitelist?

Reply:
According to [npm docs](https://docs.npmjs.com/files/package.json#files), LICENSE and README (also CHANGELOG, NOTICE, package.json, and main field) are always included.

Reply:
+1

Reply:
> the correct pattern was without dist: `src/**`

or may be not - Won is still investigating